docs/DeveloperGuide.md

```diff
@@ -444,3 +444,18 @@ testers are expected to do more *exploratory* testing.
    1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
 1. _{ more test cases …​ }_
+
+--------------------------------------------------------------------------------------------------------------------
+
+## Appendix: Planned Enhancements
+Team Size: 5
+
+1. Adding appointments that span multiple days: Currently, to add appointments that span multiple days, you can add multiple consecutive appointments for each day to cover the entire period. However, this process is tedious and time-consuming. We plan to support multi-day appointments, allowing users to add them in one step.
```

Comment:
For all the points might want to bold the main point n then line break the details so its easier to read

Reply:
I think that's a good idea. Thanks for the suggestion!